master/internal/api_config_policies.go

```diff
@@ -56,15 +57,15 @@ func (a *apiServer) validatePoliciesAndWorkloadType(
 		return configpolicy.ValidateNTSCConfig(globalConfigPolicies, configPolicies,
 			priorityEnabledErr)
 	default:
-		return status.Errorf(codes.InvalidArgument, fmt.Sprintf(invalidWorkloadTypeErr+": %s.", workloadType))
+		return fmt.Errorf(invalidWorkloadTypeErr+": %s", workloadType)
```

Comment:
isn't the workload type part of the request?
I think we may wanna leave this as `status.InvalidArgument`

Reply:
Everywhere a `validatePoliciesAndWorkloadType` err is reference handles it like
```
err = a.validatePoliciesAndWorkloadType(globalConfigPolicies, req.WorkloadType,
		req.ConfigPolicies)
	if err != nil {
		return nil, status.Errorf(codes.InvalidArgument, err.Error())
	}
```
so `InvalidArgument` is there!

Reply:
Ooh gotcha!! Great catch thanks Caroina!